src/aiida/orm/nodes/data/structure.py

```diff
@@ -1876,6 +1885,35 @@ def _get_object_pymatgen_molecule(self, **kwargs):
         positions = [list(site.position) for site in self.sites]
         return Molecule(species, positions)
 
+    def to_atomistic(self):
+        """
+        Returns the atomistic StructureData version of the orm.StructureData one.
+        """
+        if not has_atomistic():
+            raise ImportError(
```

Comment:
```suggestion
        try:
            import aiida_atomistic
        except ImportError:
            raise
```

Reply:
Hi @unkcpz, thank you for the comment.
I think `if has_atomistic()` is more clear than putting again the try/except condition? 
but do you have some specific reason to not want to use the `has_atomistic` function?

Reply:
Yes, I agree after I seeing the use as align with other atomic structure library.